template/templates.yml

```diff
@@ -0,0 +1,51 @@
+---
+# Template definitions for Spice IAs.
+
+# Each template represents one input and one output file.
+# Modification of files generated by certain templates will need to restart a
+# running duckpan instance in order to be recognized:
+#   - Perl modules
+# This is indicated by the 'needs_restart' field.
+templates:
+    pm:
+        label:  Perl Module
+        input:  lib/DDG/Spice/Example.pm
+        output: lib/DDG/Spice/<:$ia_path:>.pm
+        needs_restart: true
+
+    test:
+        label:  Perl Module Test
+        input:  t/Example.t
+        output: t/<:$ia_path:>.t
```

Comment:
Hmm although I don't think this is bad, we don't currently follow this. Right now all IA's are just `t/<package_name>.t` -- I see only 1 Spice that is nested.

Let's stick with this. I'll have the community organize the tests :+1: